pvsystemprofiler/algorithms/longitude/fitting.py

```diff
@@ -18,5 +18,5 @@ def fit_longitude(eot, solarnoon, days, gmt_offset, loss='l2'):
     cost = cost_func(sn_h[use_days] - solarnoon[use_days])
     objective = cvx.Minimize(cost)
     problem = cvx.Problem(objective)
-    problem.solve()
```

Comment:
Is this due to CVXPY changing the default solver to CLARABEL? This is a small problem, so they will both likely work (ECOS and CLARABEL) but we may want to follow the lead of CVXPY and allow clarabel to be the default.

Reply:
Yes this is due to that. I will have to update the tests using this function after switching to clarabel. I'll do that and ping you when it's ready.